__tests__/app.test.js

```diff
@@ -113,3 +113,71 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("should add a comment for an article", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "example of comprehensive and non biased comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment.body).toBe(newComment.body);
```

Comment:
Do we only expect a username and body to come back on the response from the database? Lets be more specific on asserting the response properties and values if we can.